db/seeds.rb

```diff
@@ -1,10 +1,11 @@
+
 # This file should contain all the record creation needed to seed the database with its default valuese.
 # The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
 
```

Comment:
Please add here `return unless Rails.env.development?`
And clear all data from DB or check if it exists, because error raises